app/models/project_maintenance.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ProjectMaintenance < ApplicationRecord
+  belongs_to :project
+  belongs_to :user
+
+  validates :project, :user, presence: true
+  validates :project_id, uniqueness: { scope: :user_id }
+
+  before_create :assign_position
+
+  default_scope -> { order(:position) }
+
+  private
+
+  # Add new project maintainers to the end of the list
+  def assign_position
+    self.position = if prio = self.class.where(project_id: project_id).maximum(:position)
+                      prio + 1
+                    else
+                      0
+                    end
+  end
```

Comment:
I was trying to mimic `acts_as_list` without adding another Gem dependency. 

I'm inclined to remove the position argument entirely since we don't need it at the moment. What do you think?

Reply:
Totally agree with removing it if we are not needing it right now. (*whispers YAGNI) 
💯